### ISSUE TITLE
go get

### ISSUE THREAD

**Issue by rafalohaki** [optiv/Mangle#3]
Cant install dependency

PS D:\Pobrane\Mangle> go get github.com/Binject/debug/pe
go: go.mod file not found in current directory or any parent directory.
        'go get' is no longer supported outside a module.
        To build and install a command, use 'go install' with a version,
        like 'go install example.com/cmd@latest'
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
PS D:\Pobrane\Mangle> go install github.com/Binject/debug/pe
go: 'go install' requires a version when current directory is not in a module
        Try 'go install github.com/Binject/debug/pe@latest' to install the latest version
PS D:\Pobrane\Mangle> go install github.com/Binject/debug/pe@latest
package github.com/Binject/debug/pe is not a main package
PS D:\Pobrane\Mangle>

**Comment by GeorgePatsias** [comment]
+1

**Comment by y0d4a** [comment]
do:
 go mod tidy
 and then go build Mangle.go

**Comment by gabemarshall** [comment]
The following worked for me:

```
cd Mangle && go mod init optiv/Mangle && go mod tidy
```

**Comment by sempervictus** [comment]
Just wrote up our PKGBUILD, seems to work using
```bash
build() {
  export GOPATH="$srcdir"
  cd "${srcdir}/${_pkgrname}"
  go mod init optiv/Mangle && \
  go mod tidy && \
  go build Mangle.go
}
```

**Comment by Tylous** [comment]
This has been fixed in 1.1
